### ISSUE TITLE
DJIBaseProductDelegate's productConnected() is not being called Part 2

### ISSUE THREAD

**Issue by aquilaseye** [dji-sdk/Mobile-SDK-iOS#101]
Sorry to make a duplicate issue but, this is related to Issue #100 . 

So, I updated to 4.0.1 and it is still not calling the productConnected() method or even the componentConnected() method. Since I'm using a Phantom 3 Standard which does not require a USB connection to the RC unit and connects through Wi-Fi, could this be a problem?

**Comment by XietongLU** [comment]
@aquilaseye are you using the swift sample? I tested with the Objective-C sample and it works fine. Then I check the swift sample and found that the usage may be misleading. Instead of focus on productConnected() or productDisconnected() in StartupViewController, please pay attention to ProductCommunicationManager. It uses the DJISDKManagerDelegate. Inside, you can get the connectivity change of products and components.

**Comment by aquilaseye** [comment]
@XietongLU Were you able to connect using a Phantom 3 Standard with the Swift example. Now, I cannot seem to get connected via the main DJI GO app. I may have a defective unit.

Was DJI GO updated recently?

**Comment by XietongLU** [comment]
@aquilaseye Yes. I can connect P3 Standard with Swift sample. 
If DJI GO doesn't work, it might have two reasons: 
1. There are SDK based applications running in the background. 
2. You are using DJI GO 4 but P3 Standard requires DJI GO 3.

**Comment by aquilaseye** [comment]
@XietongLU Some odd things going on. So, I uninstalled/reinstalled DJI GO. Rebooted my phone. Now the DJI GO app works and connects properly. Then, I run the DJISDKSwiftDemo and it still doesn't connect on that app. I stop that app then reopen DJI GO. Now DJI GO does not want to connect. I'll try rebooting the phone and report back here when I can connect again.

**Comment by aquilaseye** [comment]
@XietongLU I finally got it to work. I had to close the DJISDKSwiftDemo app manually on the phone rather than in XCode or else it would not properly shutdown. Another thing, most importantly, is that connection only occurs to the Phantom 3 Standard if no USB is connected. My phone was connected to the computer via USB while trying to connect and it would not work. Disconnecting _any_ USB cable from the phone allows it to connect to the drone. I'm assuming that the opposite is true for a Phantom 4 where the USB has to be connected. Good times. Thanks for the help.
